<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-T4TIAI14" failures="1" tests="5" name="Test" time="0.068" errors="0" timestamp="2024-03-01T12:04:16 IST">
  <testcase classname="com.crm.practice.DemoRetryTest" name="check" time="0.011">
    <skipped/>
  </testcase> <!-- check -->
  <testcase classname="com.crm.practice.DemoRetryTest" name="check" time="0.002">
    <skipped/>
  </testcase> <!-- check -->
  <testcase classname="com.crm.practice.DemoRetryTest" name="check" time="0.0">
    <skipped/>
  </testcase> <!-- check -->
  <testcase classname="com.crm.practice.DemoRetryTest" name="check" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.crm.practice.DemoRetryTest.check(DemoRetryTest.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- check -->
  <testcase classname="com.crm.practice.DemoRetryTest" name="finish" time="0.001"/>
</testsuite> <!-- Test -->
